### ISSUE TITLE
Change role of Accordion element in "Advanced > Attribute" section

### ISSUE THREAD

**Issue by JennCota2020** [elementor/elementor#10174]
In order to meet Accessibility Standards (WCAG 2.0 AA) I had a website audited. The website was built using Elementor Pro plugin on Wordpress with the QHub Theme (Monster Templates). One element I used often throughout the website is the "Accordion" block. 

The website underwent two audits by an external AODA Consultant to ensure it meets [AA Standards](https://www.w3.org/WAI/standards-guidelines/wcag/).

Upon inspection, the audit stated that in order to meet best practices for screen readers, the Accordion feature should have a role that reads "link" - while it currently says "tab."  

I then went to "Advance Panel" in the Accordion feature, and clicked "Attribute." Using Elementor's style guide, I tried to change the role="link": role|link

When I opened the website and inspected it, it still read as role="tab"

![TAB_FUNCTION_ROLE_WEBSITE](https://user-images.githubusercontent.com/59975486/72553823-78575e00-3867-11ea-8491-db2a9417a057.jpg)

----

**Request to fix bug:**

It would be great if we can change the role of any block in Elementor to ensure best practices are met for Screen Readers!

Hope this was informative enough.

Does anyone have any suggestions?

**Comment by rami-elementor** [comment]
@JennCota2020 

The accordion widget has `role` attributes. It uses `role=tablist` for the wrapper, `role=tab` for the tabs and `role=tabpanel` for the content.

Source: https://github.com/elementor/elementor/blob/master/includes/widgets/accordion.php#L536

Can you provide additional information why it should be `role=link` ?
